addons/rearm/functions/fnc_getAllTurrets.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: Tuupertunut
+ * Returns all turrets in a vehicle.
+ *
+ * BIS command "allTurrets" does not return the driver turret at the time of writing (2017-07-16).
+ * This function just adds driver turret to the array returned by "allTurrets".
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * Turret paths <ARRAY>
+ *
+ * Example:
+ * [vehicle] call ace_rearm_fnc_getAllTurrets
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle"];
+
+private _turrets = allTurrets _vehicle;
+
+// Adding the driver turret "[-1]".
+_turrets pushBack [-1];
```

Comment:
instead of creating new function, you could use `[[-1]] + allTurrets _vehicle` syntax, as said at [allTurrets Notes](https://community.bistudio.com/wiki/allTurrets)